pkg/channel/consolidated/utils/util.go

```diff
@@ -76,16 +77,24 @@ func GetKafkaAuthData(ctx context.Context, secretname string, secretNS string) *
 	}
 
 	kafkaAuthConfig := &client.KafkaAuthConfig{}
-	// check for TLS
+
+	// TLS
 	if string(secret.Data[TlsCacert]) != "" {
+		// We have a self-signed TLS cert
 		tls := &client.KafkaTlsConfig{
 			Cacert:   string(secret.Data[TlsCacert]),
 			Usercert: string(secret.Data[TlsUsercert]),
 			Userkey:  string(secret.Data[TlsUserkey]),
 		}
 		kafkaAuthConfig.TLS = tls
+	} else if strings.ToLower(string(secret.Data[Tls])) == strings.ToLower("enabled") {
```

Comment:
Maybe change this to a boolean named `tlsEnabled`?

/lgtm
/hold

Feel free to unhold if you think it is ok as a string.